### DISCUSSION TITLE
plugin-seo: Suggestion: better tabbedUI config

### DISCUSSION THREAD

**Discussion by hades200082** [payloadcms/payload#4587] [Feature Requests & Ideas]
Use case: I'd like to have some fields appear above my tabs in the admin UI.

If this plugin either just looks for the _first tab field_ rather than just checking if the first field is `type: "tabs"` then it could add itself to the first set of tabs it finds.

Alternatively, you could extend the plugin config to allow selection of whether the user wants the SEO tab to be added to the first main tabs field, or the first sidebar tabs field.

**Comment by PatrikKozak** [comment]
@hades200082 Appreciate the suggestion, we'll keep this in mind  - going to convert to a feature request.

**Comment by jaschaio** [comment]
Kind of related to this: Anything nested under the `Content` tab can't be used within the list view as a column. So the list view for any collection that uses the SEO plugin becomes very ugly.

**Comment by hades200082** [comment]
It might be better to just make the seo fields a custom field group that can be added where I want it on a per collection basis. 

That would fit payloads composition style approach better I think.
